src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -225,187 +235,205 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
-              </button>
-
-              <h2 id="group-creation-label" className="modal__header__title" data-uie-name="status-people-selected">
-                {selectedContacts.length
-                  ? t('groupCreationParticipantsHeaderWithCounter', selectedContacts.length)
-                  : t('groupCreationParticipantsHeader')}
-              </h2>
-
-              <button
-                className="group-creation__action enabled accent-text"
-                type="button"
-                onClick={clickOnCreate}
-                aria-label={participantsActionText}
-                data-uie-name="do-create-group"
-              >
-                {participantsActionText}
-              </button>
-            </>
-          )}
-          {stateIsPreferences && (
-            <>
-              <button
-                className="button-reset-default"
-                type="button"
-                onClick={() => setIsShown(false)}
-                aria-label={t('accessibility.groupCreationActionCloseModal')}
-                data-uie-name="do-close"
-              >
-                <Icon.Close aria-hidden="true" className="modal__header__button" />
-              </button>
-
-              <h2 id="group-creation-label" className="modal__header__title">
-                {t('groupCreationPreferencesHeader')}
-              </h2>
-
-              <button
-                id="group-go-next"
-                className={cx('group-creation__action', {
-                  'accent-text': groupNameLength,
-                  enabled: isInputValid,
-                })}
-                disabled={!isInputValid}
-                type="button"
-                onClick={clickOnNext}
-                aria-label={t('groupCreationPreferencesAction')}
-                data-uie-name="go-next"
-              >
-                {t('groupCreationPreferencesAction')}
-              </button>
-            </>
-          )}
-        </div>
-        {stateIsParticipants && (
-          <SearchInput
-            input={participantsInput}
-            setInput={setParticipantsInput}
-            selectedUsers={selectedContacts}
-            setSelectedUsers={setSelectedContacts}
-            placeholder={t('groupCreationParticipantsPlaceholder')}
-            enter={clickOnCreate}
-          />
-        )}
-
-        {stateIsParticipants && (
-          <div className="group-creation__list" ref={setScrollbarRef}>
-            {getContacts().length > 0 && (
-              <UserSearchableList
-                users={getContacts()}
-                filter={participantsInput}
-                selected={selectedContacts}
-                onUpdateSelectedUsers={setSelectedContacts}
-                searchRepository={searchRepository}
-                teamRepository={teamRepository}
-                conversationRepository={conversationRepository}
-                noUnderline
-              />
+    <StyledApp theme={theme}>
```

Comment:
```suggestion
    <StyledApp themeId={THEME_ID.DEFAULT}>
```

Reply:
Shouldn't it be `themes[THEME_ID.DEFAULT]` ? `themes` imported from ui-kit also

Reply:
you could do `theme={themes[THEME_ID.DEFAULT]}`
but we export the `THEME_ID` enum and we have a `themeId` prop

Reply:
right, didn't spot the propname change here 😄